bin/gi18n-checkup

```diff
@@ -5,14 +5,15 @@ set +e
 export GOPATH=$HOME/go
 export PATH=$PATH:$GOPATH/bin
 
-go get -u github.com/maximilien/i18n4go/gi18n
-OUTPUT=$?
-if [ $OUTPUT -ne 0 ]; then
-  printf "Cannot install latest gi18n tool to verify strings:\n${OUTPUT}"
-  exit 1
-fi
+# This doesn't compile automatically, so fails
+# go get -u github.com/maximilien/i18n4go
+# OUTPUT=$?
+# if [ $OUTPUT -ne 0 ]; then
+#   printf "Cannot install latest gi18n tool to verify strings:\n${OUTPUT}"
+#   exit 1
+# fi
 
-OUTPUT=`gi18n -c checkup`
+OUTPUT=`i18n4go -c checkup`
```

Comment:
This is a bit of a diversion, but they are using a third-party library for i18n and the command-line utility seems to have changed its name. So, it would build fine with `./bin/build` but the test script invoked by `./bin/test` would fail